src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -54,11 +73,11 @@ <h4> {% trans "Logic rules" %} </h4>
                         </thead>
                         <tbody>
                             {% for rule in log.extra_data.evaluated_rules %}
-                                    <tr>
+                                    <tr class="logic-logs__row-body logic-logs__row-body--hidden">
                                         <td>
                                             {{ rule.readable_rule }}
                                         </td>
-                                        <td>{{ rule.trigger|yesno|capfirst }} </td>
+                                        <td class="logic-logs__col-body logic-logs__col-body--trigger-{{ rule.trigger|yesno|capfirst }}">{{ rule.trigger|yesno|capfirst }} </td>
```

Comment:
I'd make this an `{% if %} ...` statement instead of relying on `yesno` which puts things through i18n, that will also make the code more predictable/readable.